wp-includes/rest-api/endpoints/class-wp-rest-customize-changesets-controller.php

```diff
@@ -457,6 +433,138 @@ public function get_items( $request ) {
 	}
 
 	/**
+	 * Checks if a given request has access to update a changeset.
+	 *
+	 * @since 4.?.?
+	 * @access public
+	 *
+	 * @param WP_REST_Request $request Full details about the request.
+	 * @return bool|WP_Error True if the request has read access for the item, WP_Error object otherwise.
+	 */
+	public function update_item_permissions_check( $request ) {
```

Comment:
@westonruter Is there a reason why it should be allowed to change the status of an existing changeset to `auto-draft`? It feels like that shouldn't be allowed. Thoughts?